src/translations/wip.wizards.json

```diff
@@ -1,96 +1,99 @@
-    "spell.wizards.fire_meteor.name": "Meteor",
-    "spell.wizards.fire_meteor.description": "Crashes a meteors on the target, each causing up to {damage} fire spell damage within {impact_range} blocks.",
+    "spell.wizards.fire_scorch.name": "Abrasar",
+    "spell.wizards.fire_scorch.description": "Abrasa al objetivo, quemándolo y causándole {damage} de hechizo de Fuego.",
+    "spell.wizards.fireball.name": "Bola de Fuego",
+    "spell.wizards.fireball.description": "Lanza una Bola explosiva de Fuego, causando hasta {damage} de hechizo de Fuego en un radio de {impact_range} bloques.",
+    "spell.wizards.fire_breath.name": "Aliento de Fuego",
+    "spell.wizards.fire_breath.description": "Incinera a los objetivos situados delante, causándoles hasta {damage} de hechizo de Fuego por segundo.",
+    "spell.wizards.fire_meteor.name": "Meteoritos",
+    "spell.wizards.fire_meteor.description": "Hace caer meteoritos sobre el objetivo, cada uno de los cuales causa hasta {damage} de hechizo de Fuego en un radio de {impact_range} bloques.",
   
-    "spell.wizards.ice_lance.name": "Ice Lance",
-    "spell.wizards.ice_lance.description": "Launches a frost shard piercing thru {pierce} target, causing {damage} frost spell damage on impact.",
-    "spell.wizards.frostbolt.name": "Frostbolt",
-    "spell.wizards.frostbolt.description": "Launches a ball of frost ricocheting to {ricochet} additional nearby targets, causing {damage} frost spell damage and slowing the target on impact.",
-    "effect.wizards.frost_slowness": "Slowness",
-    "effect.wizards.frost_slowness.description": "Reduces movement speed",
-    "spell.wizards.frost_nova.name": "Frost Nova",
-    "spell.wizards.frost_nova.description": "Freezes targets around you for {effect_duration} seconds, causing {damage} frost spell damage and blocking their movement. Frozen targets are vulnerable to frost magic.",
-    "effect.wizards.frozen": "Frozen",
-    "effect.wizards.frozen.description": "Prevents movement, removed upon taking damage, vulnerable to frost magic",
-    "spell.wizards.frost_shield.name": "Frost Shield",
-    "spell.wizards.frost_shield.description": "Protects you from attacks, projectiles and fire for {effect_duration} seconds, but also slows down your movement.",
-    "effect.wizards.frost_shield": "Frost Shield",
-    "effect.wizards.frost_shield.description": "Blocks incoming attacks while active, but slows down movement",
+    "spell.wizards.ice_lance.name": "Lanza de Escarcha",
+    "spell.wizards.ice_lance.description": "Lanza una esquirla de escarcha que atraviesa {pierce} objetivos, causando {damage} de hechizo de Escarcha al impactar.",
+    "spell.wizards.frostbolt.name": "Descarga Gélida",
+    "spell.wizards.frostbolt.description": "Lanza una bola de escarcha que rebota hacia {ricochet} objetivos cercanos adicionales, causando {damage} de hechizo de Escarcha y ralentizando al objetivo al impactar.",
```

Comment:
Lo mismo,

"Causando 18 de hechizo de escarcha"

No suena muy bien in-game y puede ser confuso ya que no menciona específicamente que es daño